README.md

```diff
@@ -0,0 +1,111 @@
+Alive Language Server
+=====================
+
+This is the language server for [Alive: The Average Lisp VSCode Environment](https://github.com/nobody-famous/alive).
+
+## Running the server
+
+For newcomers it could be difficult the start with Common LISP specially if the tech stack is not known yet. This instructions helps to have the server up and running as easy as possible. Steps are:
+
+1. Install your LISP Implementation
+2. Install a library/package manager
+3. Clone this repository
+4. Load the `alive-lsp` project and start the server
+
+### Install your LISP Implementation
+
+Here lets assume we install [sbcl](https://www.sbcl.org/)
+
+```bash
+# debian
+apt-get install sbcl
+
+# macos
+brew install sbcl
+```
+
+### Install a library/package manager
+
+Here lets assume we use [quicklisp](https://www.quicklisp.org/beta/)
+
+```bash
+curl -O https://beta.quicklisp.org/quicklisp.lisp
+curl -O https://beta.quicklisp.org/quicklisp.lisp.asc
+gpg --verify quicklisp.lisp.asc quicklisp.lisp
+sbcl --load quicklisp.lisp
+# ....
+* (quicklisp-quickstart:install)
+# it is also advisable to:
+* (ql:add-to-init-file)
+```
+
+### Clone this repository
+
+Previous step should have created a directory in your home directory: `~/quicklisp`, unless you decided to install it in another directory
+```
+(quicklisp-quickstart:install :path "~/.quicklisp")
+```
+just keep in mind the directory, lets called here `$QUICKLISP_HOME`. Lets clone this repository in there in order to make it easy to load it as project:
```

Comment:
"Let's clone Alive's repository in Quicklisp's local-projects directory, so that it will be discovered automatically by Quicklisp." (or similar)